AdobeStockAssetApi/Model/Asset/Command/LoadByIdsInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockAssetApi\Model\Asset\Command;
+
+use Magento\AdobeStockAssetApi\Api\Data\AssetInterface;
+
+/**
+ * Used to load Adobe Stock assets for gotten asset ids.
+ */
+interface LoadByIdsInterface
+{
+    /**
+     * Load Adobe Stock assets by ids
+     *
+     * @param int[] $ids
+     *
+     * @return AssetInterface[]
+     */
+    public function execute(array $ids): array;
```

Comment:
@coderimus should we use the bulk approach to all of the Asset commands?

Reply:
@sivaschenko I think no, we should not practise the bulk approach for all Asset related commands. From my point of view, using it everywhere is overhead and, personally, breaks the single responsibility approach. I know that this sounds like a not so good example, but in fact, it is :) When you need to get one item by its id it doesn't seem to be logical to get it with the operation for getting multiple items. We even have different conditions in the SQL (equal and in).

In conclusion: I left the LoadById command and introduced the new one - MultiplyLoadById. We need such functionality at the AppendAttributes class.